_episodes/55.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: episode
+title: "55: Commander Bun Bun peeks at Peaka"
+date: 2024-01-18
+tags: trino peaka
+youtube_id: "2NrgTvJP4Lc"
+wistia_id: "4qdxt6wsgf"
+sections:
+- title: "Intro"
+  desc: "Episode 55: Commander Bun Bun peeks at Peaka"
+  time: 0
+- title: "Trino releases"
+  desc: "437-438"
+  time: 96
+- title: "Concept of the episode"
+  desc: "Introducing Peaka"
+  time: 252
+- title: "An overview of Peaka"
+  desc: "What it does and how it works"
+  time: 487
+- title: "Engineering of Peaka"
+  desc: "Architecture and under the hood"
+  time: 962
+- title: "Peaka's extra Trino connectors"
+  desc: "How they're implemented and how they work"
+  time: 1204
+- title: "Demo of the episode"
+  desc: "Trying out Peaka"
+  time: 1611
+- title: "Managing catalogs and security"
+  desc: "One cluster, many catalogs"
+  time: 2494
+- title: "Peaka wrap-up"
+  desc: "Where to find it"
+  time: 3066
+- title: "PR of the episode"
+  desc: "Filesystem caching with Alluxio"
+  time: 3194
+- title: "Rounding out"
+  desc: "Outro, upcoming TCBs, and Trino Fest looms..."
+  time: 3376
+---
+
+## Hosts
+
+* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
+  at [Starburst](https://starburst.io)
+* Manfred Moser, Director of Technical Content at
+  [Starburst](https://starburst.io)
+  ([@simpligility](https://twitter.com/simpligility))
+
+## Guests
+
+* [Mustafa Sakalsiz](https://linkedin.com/in/sakalsiz), CEO at
+  [Peaka](https://www.peaka.com/)
+* [Ali Tekin](https://www.linkedin.com/in/alitekin/), Principal Software
+  Architect at [Peaka](https://www.peaka.com/)
+
+## Releases 437-438
+
+[Trino 437](https://trino.io/docs/current/release/release-437.html)
+
+* Support for configuring compression codecs
+* Support for `char` values in the `to_utf8()` and `lpad()` functions
+* Improved performance for Delta Lake queries without table statistics
+* Improved performance for Iceberg queries with filters on `ROW` columns
+
+[Trino 438](https://trino.io/docs/current/release/release-438.html)
+
+* Support for access control with [Open Policy Agent](https://trino.io/blog/2024/02/06/opa-arrived)
+* Support for `ALTER COLUMN ... DROP NOT NULL` in Iceberg and PostgreSQL
+* Support for configuring page sizes in Delta Lake, Hive, and Iceberg
+* Better type support for the `reduce_agg()` function
+* [Trino Gateway version 5](https://github.com/trinodb/trino-gateway/blob/main/docs/release-notes.md#trino-gateway-5-24-jan-2024)
```

Comment:
This needs to move to a separate sentence or so after the 438 section. Its not related to 438